readme.md

```diff
@@ -65,7 +76,7 @@ Describe the purpose of views and models in FE JS.
 
 Your Answer:
 ```text
-
+Models hold information, but typically don’t handle behaviour. Views format how the information is formatted.
```

Comment:
Front-end models represent our data, and their actions. Generally, they contain methods for CRUD actions. In order to persist to the DB, a Front-End model will usually use AJAX requests to update the data accordingly on the back end.

Front-end views generally have two main purposes:
1. Take data (from F.E. Models) and render them to HTML and put them on the page (DOM).
2. Defining how to respond to user events (clicks, keypresses, etc), by updating the DOM, using models, etc.

Because they have two 'jobs', they are sometimes called ViewControllers.